unit_test/fuzzing/test_secured_message/test_spdm_encode_secured_message/spdm_encode_secured_message.c

```diff
@@ -57,12 +57,12 @@ void libspdm_test_encode_secured_message(void **State)
                              32; /* MCTP_MAX_RANDOM_NUMBER_COUNT */
     LIBSPDM_ASSERT(spdm_test_context->test_buffer_size > record_header_max_size);
 
-    secured_message_size = spdm_test_context->test_buffer_size - record_header_max_size;
+    secured_message_size = sizeof(secured_message);
```

Comment:
Please explain why.

Reply:
uint8_t secured_message[LIBSPDM_MAX_MESSAGE_BUFFER_SIZE];
secured_message requires a section of memory, which cannot be less than the total_secured_message_size
https://github.com/DMTF/libspdm/blob/345e2f866ca463f989343a126e7beb6a563d7c31/library/spdm_secured_message_lib/libspdm_secmes_encode_decode.c#L252-L255